pkg/api/customization/cluster/actions_cis.go

```diff
@@ -0,0 +1,62 @@
+package cluster
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
+	"github.com/sirupsen/logrus"
+	"net/http"
+	"time"
+
+	"github.com/rancher/norman/httperror"
+	"github.com/rancher/norman/types"
+	"github.com/rancher/rancher/pkg/controllers/user/cis"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (a ActionHandler) runCISScan(actionName string, action *types.Action, apiContext *types.APIContext) error {
+	cluster, err := a.ClusterClient.Get(apiContext.ID, v1.GetOptions{})
```

Comment:
maybe someone else can confirm too, but we normally use access.ByID in action handlers, that also makes sure of auth vs clusterClient just getting from db

Reply:
will fix